Dockerfile

```diff
@@ -39,119 +39,122 @@ COPY --from=builder /tmp/bitgo/modules/express /var/bitgo-express/
 
 #COPY_START
 COPY --from=builder /tmp/bitgo/modules/sdk-core /var/modules/sdk-core/
-RUN cd /var/modules/sdk-core && yarn link
 COPY --from=builder /tmp/bitgo/modules/bls-dkg /var/modules/bls-dkg/
-RUN cd /var/modules/bls-dkg && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-lib-mpc /var/modules/sdk-lib-mpc/
-RUN cd /var/modules/sdk-lib-mpc && yarn link
 COPY --from=builder /tmp/bitgo/modules/statics /var/modules/statics/
-RUN cd /var/modules/statics && yarn link
 COPY --from=builder /tmp/bitgo/modules/utxo-lib /var/modules/utxo-lib/
-RUN cd /var/modules/utxo-lib && yarn link
 COPY --from=builder /tmp/bitgo/modules/blake2b /var/modules/blake2b/
-RUN cd /var/modules/blake2b && yarn link
 COPY --from=builder /tmp/bitgo/modules/blake2b-wasm /var/modules/blake2b-wasm/
-RUN cd /var/modules/blake2b-wasm && yarn link
 COPY --from=builder /tmp/bitgo/modules/bitgo /var/modules/bitgo/
-RUN cd /var/modules/bitgo && yarn link
 COPY --from=builder /tmp/bitgo/modules/abstract-utxo /var/modules/abstract-utxo/
-RUN cd /var/modules/abstract-utxo && yarn link
 COPY --from=builder /tmp/bitgo/modules/blockapis /var/modules/blockapis/
-RUN cd /var/modules/blockapis && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-api /var/modules/sdk-api/
-RUN cd /var/modules/sdk-api && yarn link
 COPY --from=builder /tmp/bitgo/modules/sjcl /var/modules/sjcl/
-RUN cd /var/modules/sjcl && yarn link
 COPY --from=builder /tmp/bitgo/modules/unspents /var/modules/unspents/
-RUN cd /var/modules/unspents && yarn link
 COPY --from=builder /tmp/bitgo/modules/account-lib /var/modules/account-lib/
-RUN cd /var/modules/account-lib && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-algo /var/modules/sdk-coin-algo/
-RUN cd /var/modules/sdk-coin-algo && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-atom /var/modules/sdk-coin-atom/
-RUN cd /var/modules/sdk-coin-atom && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-avaxc /var/modules/sdk-coin-avaxc/
-RUN cd /var/modules/sdk-coin-avaxc && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-avaxp /var/modules/sdk-coin-avaxp/
-RUN cd /var/modules/sdk-coin-avaxp && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-eth /var/modules/sdk-coin-eth/
-RUN cd /var/modules/sdk-coin-eth && yarn link
 COPY --from=builder /tmp/bitgo/modules/sdk-coin-bld /var/modules/sdk-coin-bld/
-RUN cd /var/modules/sdk-coin-bld && yarn link
```

Comment:
Was this intentional?

Reply:
If so the script is going to either bork on this file, or possibly create duplicates. Recommend testing changes with script updates.

Reply:
Yes, I was running into a persistent `max depth exceeded` error on the docker-build step, which is caused by too many layers. Consolidating the run commands should fix this, but I'm not super experienced with docker to be sure the change should work the same.

Reply:
sorry, what do you mean by "script updates"?

Reply:
There is a script that manages this file. `scripts/update-dockerfile`

Reply:
okay updated script to do the same thing